common/djangoapps/track/tests/test_middleware.py

```diff
@@ -31,15 +31,22 @@ def test_normal_request(self):
 
     def test_request_with_latin1_characters(self):
         """
-        When HTTP_USER_AGENT in request.META contains latin1 characters.
+        When HTTP headers contains latin1 characters.
         """
         request = self.request_factory.get('/somewhere')
+        # pylint: disable=no-member
         request.META['HTTP_USER_AGENT'] = 'test latin1 \xd3 \xe9 \xf1'  # pylint: disable=no-member
+        request.META['PATH_INFO'] = 'test latin1 \xd3 \xe9 \xf1'
```

Comment:
this pattern looks like a perfect candidate for `ddt` — convert this test to use that?